scripts/ci_setup.py

```diff
@@ -242,6 +272,7 @@ def __main(args: list) -> int:
 
         with open(args.output_file, 'w') as out_file:
             out_file.write(which)
+            out_file.write(pgo_config)
```

Comment:
this needs to use the variable format, right?

Reply:
Yes, good catch.